x/consensus/go.mod

```diff
@@ -0,0 +1,174 @@
+module cosmossdk.io/x/consensus
+
+go 1.22.2
+
+require (
+	cosmossdk.io/api v0.7.4
+	cosmossdk.io/collections v0.4.0
+	cosmossdk.io/core v0.12.1-0.20231114100755-569e3ff6a0d7
+	cosmossdk.io/depinject v1.0.0-alpha.4
+	cosmossdk.io/log v1.3.1
+	cosmossdk.io/store v1.1.0
+	cosmossdk.io/x/auth v0.0.0-00010101000000-000000000000
+	github.com/cometbft/cometbft v0.38.7-0.20240412124004-1f67e396cf45
+	github.com/cosmos/cosmos-proto v1.0.0-beta.5
+	github.com/cosmos/cosmos-sdk v0.51.0
```

Comment:
is this permissible? for x/conesnesus to depend the sdk.. what in the sdk module does it need?

Reply:
It's used in tests and a few other places. We need to spend some time undoing the sdk import before final release.